CHANGES

```diff
@@ -9,6 +9,17 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) X509 certificates signed using SHA1 are no longer allowed at security
+     level 1, the highest security level it's accepted at is 0.
+     When used in TLS/SSL the default security level is 1. It can be set either
+     using the cipher string with @SECLEVEL, or calling
+     SSL_CTX_set_security_level(). A call to SSL_CTX_use_certificate() will
+     fail if the security level is not lowered first.
+     Outside TLS/SSL, the default security level is -1. It can be set using
```

Comment:
Add (effectively 0). (The only difference relates to how verify parameter inheritance works, -1 means not set, while 0 is explicitly 0, so when inheritance only picks up new settings 0 stays unchanged, but -1 picks up new values from the default object.  Most users don't need to know or care, so you could just say "0").